invenio_records_marc21/templates/invenio_records_marc21/landing_page/record.html

```diff
@@ -68,10 +68,42 @@
         {%- endblock record_title -%}
 
         {%- block record_files -%}
-        <div class="top-padded"></div>
-        <hr class="thin-line">
-        {%- include "invenio_records_marc21/landing_page/helpers/files.html" %}
-        {%- endblock record_files -%}
+              <div class="ui segment" style="display: none;">
+                Debug info:
+                files.enabled: {{ record.files.enabled }}
+                access.files: {{ record.access.files }}
+                permissions.can_read_files: {{ permissions.can_read_files }}
+                is_authenticated: {{ current_user.is_authenticated }}
+              </div>
+
+              <div class="top-padded"></div>
+              <hr class="thin-line">
+
+              {% if record.access.files == 'restricted' %}
```

Comment:
i think it would be better to move those changes to the `invenio_records_marc21/landing_page/helpers/files.html` file, so we can encapsulate the file display behavior there.

and please indent the files.html files correctly before you do the change! it looks horrible how it is!

Reply:
please open this in vscode!